src/utils/daemon.rs

```diff
@@ -220,38 +223,44 @@ pub fn start_daemon(background: bool, config: Arc<Config>) -> Result<(), Error>
     }).unwrap();
 
     // update release activity everyday at 23:55
-    thread::Builder::new()
-        .name("release activity updater".to_string())
-        .spawn(move || loop {
-            thread::sleep(Duration::from_secs(60));
-            let now = Utc::now();
-
-            if now.hour() == 23 && now.minute() == 55 {
-                info!("Updating release activity");
-                if let Err(e) = update_release_activity() {
-                    error!("Failed to update release activity: {}", e);
-                }
-            }
-        })
-        .unwrap();
+    let cloned_db = db.clone();
```

Comment:
Why do each of these threads need a separate `cloned_db`?

Reply:
Because that instance of the pool is moved in each closure.

Reply:
Oh I see, this is https://github.com/rust-lang/rfcs/issues/2407. No need to change anything.